app/Http/Requests/PostRequest.php

```diff
@@ -29,7 +29,9 @@ public function rules()
             'campaign_id' => 'required|integer',
             'northstar_id' => 'nullable|objectid',
             'type' => 'required|string|in:photo,voter-reg,text,share-social',
+            // @TODO: eventually, deprecate action in the payload and make action_id required when all systems have been updated.
             'action' => 'required|string',
```

Comment:
Is this the `name` that you referred to here?

> When a post is created, it grabs the action_id from the payload. If this is not included in the payload, it goes into the database and looks for this action_id, found by querying the actions table by campaign_id, post_type, and name (all info that should come in with the post request).

Reply:
Yes exactly!

Reply:
thanks!